torch/_prims/__init__.py

```diff
@@ -1816,6 +1818,69 @@ def _item_meta(a: TensorLikeType) -> TensorMeta:
     doc=_item_doc,
 )
 
+# NOTE: need to model meta scalars
+# See https://github.com/pytorch/pytorch/issues/78070
+def _maximum_value_meta(dtype: torch.dtype) -> TensorMeta:
+    number_type = utils.dtype_to_type(dtype)
+    return TensorMeta(number_type(-1))
+
+
+def _maximum_value_aten(dtype: torch.dtype):
+    if dtype == torch.bool:
+        return True
+    elif dtype.is_floating_point:
```

Comment:
Does this return true for complex dtypes, too?

Reply:
Weird. finfo works with complex dtypes but it isn't a floating-point type.

Reply:
Yeah... we decided to add complex dtypes to finfo instead of creating a separate cinfo